Frontend/Pages/UserManagement/Register.razor

```diff
@@ -83,14 +87,28 @@
     </div>
 </div>
 
-@* TODO implement registration logic *@
 
 @code {
     readonly Registration _registration = new();
 
-    private void HandleValidSubmit() {
-        Console.WriteLine("Valid");
-        Console.WriteLine(_registration.Email);
+    private async void HandleValidSubmit() {
+        var request = new HttpRequestMessage(HttpMethod.Post,
+            "https://localhost:7038/api/register");
+
+        request.Content = new StringContent(JsonConvert.SerializeObject(_registration), Encoding.UTF8, "application/json");
+
+        var client = ClientFactory.CreateClient();
```

Comment:
Why `IHttpClientFactory`? We're already injecting a scoped `HttpClient` service, so either we should stick with that or remove it. @JohHub it's your scoped service, any pointers?

Reply:
This seems to be a mixture of the named client and the typed client approach. (See: https://docs.microsoft.com/en-us/aspnet/core/fundamentals/http-requests?view=aspnetcore-6.0#named-clients) I'd suggest the typed client approach since we are not requiring distinct uses of HttpClient because we only send requests to our backend. I'd also suggest refractoring the HttpClient into a Service, calling its methods here (See: https://docs.microsoft.com/en-us/aspnet/core/fundamentals/http-requests?view=aspnetcore-6.0#typed-clients and https://docs.microsoft.com/en-us/aspnet/core/blazor/call-web-api?view=aspnetcore-6.0&pivots=webassembly#typed-httpclient)